Documentation/network/servicemesh/envoy-load-balancing.rst

```diff
@@ -0,0 +1,116 @@
+services using Cilium, which is useful for some use cases such as gRPC
+load-balancing.
+
+Deploy Test Applications
+========================
+
+.. parsed-literal::
+
+    $ kubectl apply -f \ |SCM_WEB|\/examples/kubernetes/servicemesh/envoy/test-application-proxy-loadbalancing.yaml
+
+The test workloads consist of:
+
+- one client deployment ``client``
+- one service ``echo-service`` with two backend pods.
+
+View information about these pods:
+
+.. code-block:: shell-session
+
+    $ kubectl get pods --show-labels -o wide
+    NAME                              READY   STATUS    RESTARTS   AGE    IP          NODE           NOMINATED NODE   READINESS GATES   LABELS
+    client-7dccb64ff6-t5gc7         1/1     Running   0          39s   10.244.0.125   minikube   <none>           <none>            kind=client,name=client,pod-template-hash=7dccb64ff6
+    echo-service-744b6dd45b-487tn   2/2     Running   0          39s   10.244.0.71    minikube   <none>           <none>            kind=echo,name=echo-service,other=echo,pod-template-hash=744b6dd45b
+    echo-service-744b6dd45b-mdjc2   2/2     Running   0          39s   10.244.0.213   minikube   <none>           <none>            kind=echo,name=echo-service,other=echo,pod-template-hash=744b6dd45b
+
+.. code-block:: shell-session
+
+    $ export CLIENT=$(kubectl get pods -l name=client -o jsonpath='{.items[0].metadata.name}')
+
+Start Observing Traffic with Hubble
+===================================
+
+Enable Hubble in your cluster with the step mentioned in :ref:`hubble_setup`.
+
+Start a second terminal, then enable hubble port forwarding and observe
+traffic for the service ``echo-service``:
+
+.. code-block:: shell-session
+
+    $ kubectl -n kube-system port-forward deployment/hubble-relay 4245:4245 &
+    $ hubble observe --service echo-service -f
+
+
+You should be able to get a response from both of the backend services
+individually from ``client2``:
+
+.. code-block:: shell-session
+
+    $ kubectl exec -it $CLIENT -- curl -v echo-service:8080/
+
+Notice that Hubble shows all the flows between client pod and the backend pods
+via ``echo-service`` service.
+
+::
+
+    Jan 16 04:28:10.690: default/client-7dccb64ff6-t5gc7 (ID:5152) <> default/echo-service:8080 (world) pre-xlate-fwd TRACED (TCP)
+    Jan 16 04:28:10.690: default/echo-service:8080 (world) <> default/client-7dccb64ff6-t5gc7 (ID:5152) post-xlate-rev TRANSLATED (TCP)
+
+Add Proxy Load Balancing annotations to the services
+====================================================
+
+Adding a Layer 7 policy introduces the Envoy proxy into the path for this traffic.
+
+.. parsed-literal::
+
+    $ kubectl annotate service echo-service io.cilium.service/lb-l7=enabled
+    service/echo-service annotated
+
+Make a request to a backend service and observe the traffic with Hubble again:
+
+.. code-block:: shell-session
+
+    $ kubectl exec -it $CLIENT -- curl -v echo-service:8080/
+
+::
+
```

Comment:
```suggestion
```

No need to split the command and its output, `shell-session` means we have both in the block.

Reply:
ah the output is mainly for hubble observe command but not curl command, I have added one line in between to make it clearer.